.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+on: push
+name: Build
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ['2', '3']
```

Comment:
Can we not be more specific about versions here like we were being on Travis?

Reply:
Sure. I just thought it might be better to say that we want to always test against the latest ruby version. If not, the matrix will grow, but it's not always beneficial. For example why do we need to test against ruby `2.7.3` and ruby `2.7.4`. Maybe we could just test agains the latest in the minor version field, e.g. `2.7.x` and `3.1.x`?

Reply:
> Maybe we could just test agains the latest in the minor version field, e.g. `2.7.x` and `3.1.x`?

Yes, that's fine, but we do typically want to test multiple of those minor versions.